helper-cli/src/test/assets/path-exclude-gen/expected-excluded-dirs.txt

```diff
@@ -0,0 +1,981 @@
+.github/**
+.idea/**
+.mvn/**
+.teamcity/.mvn/**
+.teamcity/src/test/**
```

Comment:
There's a lot of `/src/test/` paths in here... how about first changing the path exclude generator to exclude the `**/src/test/**` pattern?

Reply:
If at all, I wanted to do patterns starting with `**/` in a later state, making it toggle-able by the user and configurable by a threshold. So, then it still makes sense to have a test with that option disabled. Note: The impact of not having `**/` in most of the cases should be small, as the generation is supposed to be run only against one package at a time, so against way less file paths.